src/main/java/com/blamejared/crafttweaker/impl/item/MCItemStack.java

```diff
@@ -132,4 +132,21 @@ public String toString() {
     public IItemStack[] getItems() {
         return new IItemStack[] {this};
     }
+    
+    @Override
+    public boolean equals(Object o) {
+        if(this == o)
+            return true;
+        if(o == null || getClass() != o.getClass())
+            return false;
+        
+        MCItemStack that = (MCItemStack) o;
+    
+        return internal.equals(that.internal);
```

Comment:
Is this meant to compare the item stacks beyond object equality, given ItemStack does not override equals *or* hashCode